sdks/python/apache_beam/transforms/stats_test.py

```diff
@@ -154,13 +155,15 @@ def test_get_sample_size_from_est_error(self):
 
   @unittest.skipIf(sys.version_info < (3, 0, 0),
```

Comment:
If this test passes on Python 2, consider unskipping it on Python 2. Note that on Python 2 hash is deterministic, so I think we won't have problem with flakiness.

Reply:
Yes, with fixed input and deterministic hash function, it wouldn't fail. I will undo skipping.

Reply:
This test failed again with py27, so I added skip back.